packages/functional-tests/src/functional-tests.local.spec.ts

```diff
@@ -0,0 +1,159 @@
+    const clientSecret = process.env.SP_PASSWORD;
+    const tenantId = process.env.SP_TENANT;
+    const apimName = process.env.APIM_SERVICE_NAME;
+    const cosmosKey = process.env.COSMOS_DB_KEY;
+    const cosmosUrl = process.env.COSMOS_DB_URL;
+
+    let logger: GlobalLogger;
+    let a11yServiceClient: A11yServiceClient;
+    let testRunner: TestRunner;
+    let guidGenerator: GuidGenerator;
+    let testContextData: TestContextData;
+    let onDemandPageScanRunResultProvider: OnDemandPageScanRunResultProvider;
+    let releaseIdStub: string;
+    let runIdStub: string;
+
+    beforeAll(async () => {
+        if (isServiceCredProvided()) {
+            const container = new Container({ autoBindInjectable: true });
+            setupRuntimeConfigContainer(container);
+            container.bind(Logger).toDynamicValue(_ => {
+                return new GlobalLogger([new ConsoleLoggerClient(container.get(ServiceConfiguration), console)], process);
+            });
+            registerAzureServicesToContainer(container, CredentialType.AppService);
+            const cred = new A11yServiceCredential(clientId, clientSecret, clientId, `https://login.microsoftonline.com/${tenantId}`);
+            a11yServiceClient = new A11yServiceClient(cred, `https://apim-${apimName}.azure-api.net`);
+            onDemandPageScanRunResultProvider = container.get(OnDemandPageScanRunResultProvider);
+            guidGenerator = container.get(GuidGenerator);
+            logger = container.get(Logger);
+            testRunner = container.get(TestRunner);
+            await logger.setup({
+                source: 'dev box',
+            });
+            testRunner.setLogger(logger);
+            testContextData = {
+                scanUrl: 'https://www.washington.edu/accesscomputing/AU/before.html',
+            };
+            releaseIdStub = `dev-${guidGenerator.createGuid()}`;
+            runIdStub = `dev-${guidGenerator.createGuid()}`;
+        } else {
+            console.log('One or more service credentials is missing. Skipping All functional tests.');
+        }
+    });
+
+    testIf('PostScan', isServiceCredProvided, async (done: jest.DoneCallback) => {
+        const postScanTests = getTests('PostScan');
+        await testRunner.run(postScanTests, TestEnvironment.all, releaseIdStub, runIdStub);
+        done();
+    });
+
+    testIf('ScanStatus', isServiceCredProvided, async (done: jest.DoneCallback) => {
+        const scanStatusTests = getTests('ScanStatus');
+        testContextData.scanId = (await a11yServiceClient.postScanUrl(testContextData.scanUrl)).body[0].scanId;
+        await testRunner.run(scanStatusTests, TestEnvironment.all, releaseIdStub, runIdStub);
+        done();
+    });
+
+    testIf('ScanExecution', isServiceCredProvided, async (done: jest.DoneCallback) => {
+        const scanRunResultResponse = await waitForScanRequestCompletion();
+        expect(scanRunResultResponse.reports.length).toEqual(2);
```

Comment:
Should this function just run e2e tests w/o any validation? If we need validation it should be part of an e2e test instead

Reply:
removed.